### ISSUE TITLE
ReLU instead of sigmoid

### ISSUE THREAD

**Issue by Estalhun** [julius-speech/julius#95]
Dear Lee,

I would like to use TensorFlow (Keras) to make a ANN with ReLU activation.
I found the sigmoid calculation (d = 1.0 / (1.0 + exp(-x));) but I am not sure at all to modify this line to d = x>0 ? x : 0; will solve the problem.

Do you have plans to implement ReLU into Julius?

Cheers,
István

**Comment by nitslp-ri** [comment]
Hi,

Yes, ReLU can be easily implemented in Julius like the way you wrote.
